tutorials/quickstart/Solution/Platform/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Platform
+{
+    using System;
+
+    class Program
+    {
+        #region PlatformMain
+
+        static void Main(string[] args)
+        {
+            Console.Title = "Particular Service Platform Launcher";
+            //Particular.PlatformLauncher.Launch();
```

Comment:
Why is this commented?

Reply:
The Launch statement for the platform sample is commented out because the text doesn't guide toward using the platform until about halfway down.